pkg/reconciler/ingress/ingress_test.go

```diff
@@ -248,8 +248,8 @@ func TestReconcile(t *testing.T) {
 			},
 		},
 		WantCreates: []runtime.Object{
-			resources.MakeMeshVirtualService(insertProbe(ing("no-virtualservice-yet", 1234)), gateways),
-			resources.MakeIngressVirtualService(insertProbe(ing("no-virtualservice-yet", 1234)),
+			resources.MakeMeshVirtualService(context.Background(), insertProbe(ing("no-virtualservice-yet", 1234)), gateways),
```

Comment:
May be extract & use something like

```
	ctx := config.ToContext(TestContextWithLogger(t), ReconcilerTestConfig())
```

Since `getGatewayURLFromContext()` uses a `config.Istio` object, which `ReconcilerTestConfig()` already provides.

Reply:
none of these tests actually do anything with the config.Istio (that's used and tested in reosurces/virtual_service[_test].go) so it seems like a bit more noise than it's worth here. Will happily add if you feel strongly though.